packages/botonic-react/src/components/multichannel/multichannel-facebook.jsx

```diff
@@ -0,0 +1,39 @@
+import { MAX_CHARACTERS_FACEBOOK } from './multichannel-utils'
+
+export class MultichannelFacebook {
+  constructor() {}
+
+  convertText(props, originalText) {
+    if (originalText.length > MAX_CHARACTERS_FACEBOOK) {
+      const { texts, finalText } = this.splitText(originalText)
+      const newProps = this.getNewProps(props, finalText)
+      return { texts, newProps }
+    }
+    return { newProps: props }
+  }
+
+  getNewProps(props, finalText) {
+    const newProps = { ...props }
+    Array.isArray(props.children)
+      ? (newProps.children[0] = finalText)
```

Comment:
beware this kind of logic (accessing directly with children[0]). In my opinion, it will be better to filter children by elements that are not React Components and then use the operator `join('')` to reagroup them.
You have an example in webchat's `text.jsx`.
```
const text = children
    .filter(e => !e.type) // <-- This
    .map(e => {
      if (Array.isArray(e)) return serializeText(e)
      else return String(e)
    })
    .join('')
```